EasyPost/Http/Request.cs

```diff
@@ -1,37 +1,56 @@
+using System;
 using System.Collections.Generic;
+using System.Net.Http;
+using System.Text;
 using EasyPost._base;
 using EasyPost.Utilities.Internal;
-using RestSharp;
-using RestSharp.Serializers;
 
 namespace EasyPost.Http
 {
+    // TODO: Make disposable and dispose of the request message
```

Comment:
question: why not do this now?

Reply:
Going to in a separate PR. Gotta stay focused 😉

Reply:
My my, I may cry 😂